### PR TITLE
Bug 1170464 - [SMS] fix remain jsdoc error

### PR DESCRIPTION
**Pull request by Gioyik**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=233d0b58955a037edf8f2afa2d8b558424a5f94f

File: apps/sms/views/shared/js/utils.js [hunk 1]
```diff
@@ -711,7 +711,7 @@
        * flow is paused until yielded Promise is resolved, so that consumer gets
        * Promise result instead of Promise instance itself.
        * See https://www.promisejs.org/generators/ as the reference.
-       * @param {function*} generatorFunction Generator function that yields
+       * @param {*} generatorFunction Generator function that yields
```
Comment (julienw) [inline apps/sms/views/shared/js/utils.js]: here we want to express we have a generator function as parameter (a function that, when called, returns an iterator).

Maybe `{function(...*): Iterator}\ ?

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=65949ab56f2c318f63bab311ef9d95f2cf2a238d

File: apps/sms/views/shared/js/utils.js (hunk 1 above)
Comment (gasolin) [inline apps/sms/views/shared/js/utils.js]: :+1:

File: apps/sms/views/shared/js/utils.js [hunk 2]
```diff
@@ -711,7 +711,7 @@
        * flow is paused until yielded Promise is resolved, so that consumer gets
        * Promise result instead of Promise instance itself.
        * See https://www.promisejs.org/generators/ as the reference.
-       * @param {function*} generatorFunction Generator function that yields
+       * @param {function(...*): Iterator} generatorFunction Generator function that yields
```
Comment (gasolin) [inline apps/sms/views/shared/js/utils.js]: LINT error: apps/sms/views/shared/js/utils.js: line 830, col 91, Line is too long. (ERROR)

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=96d6fa0ccf59faffd5fa96e58a03c2b65a47f837

**Pull request merged**